### PR TITLE
Fix: Added condition in Inprogress email handlebars template to better format informantName in email

### PR DESCRIPTION
**Pull request by tareq89**
https://github.com/opencrvs/opencrvs-core/issues/5744

### CODE REVIEW

File: src/api/notification/email-templates/marriage/registration.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
                 max-width: 100%;">
     <h1>Marriage registration completed</h1>
     <p>
-        Hello {{informantName}},
+        Hello{{formatName informantName}},
     </p>
     <p>
         Congratulations, the marriage of {{groomName}} and {{birdeName}} has been registered.
```
Comment (rikukissa) [inline src/api/notification/email-templates/marriage/registration.html:43]:
```
Congratulations, the marriage of {{groomName}} and {{birdeName}} has been registered.
```

`birdeName` must be a bug here as well

**Review by rikukissa — CHANGES_REQUESTED** [review]
@tareq89 what is the root cause for the issue mentioned in the ticket? What happens if you do not add a helper for formatting an undefined value?

What if one of the other values are undefined? There must be a more generic fix for all missing values to be rendered a certain way. Empty string is fine but I think handlebars does it by default so a helper is not needed.

**Comment by tareq89** [comment]
> @tareq89 what is the root cause for the issue mentioned in the ticket? What happens if you do not add a helper for formatting an undefined value?
> 
> What if one of the other values are undefined? There must be a more generic fix for all missing values to be rendered a certain way. Empty string is fine but I think handlebars does it by default so a helper is not needed.
Screenshot 2024-06-24 at 2 21 26 PM

When informant name is null, there is an extra space in between Hello and comma (,). According to recommendation by [SyedaAfrida](https://github.com/SyedaAfrida), the issue is handled in this way.

File: src/api/notification/email-templates/marriage/registration.html (hunk 1 above)
Comment (tareq89) [reply src/api/notification/email-templates/marriage/registration.html:43]: I see, I think we should test out all the email templates with empty value to find out what the users experiences in corner cases. On it.

Comment (tareq89) [reply src/api/notification/email-templates/marriage/registration.html:43]: On another note, only in the in progress email, there is a chance that values can be null. And in progress emails have only one variable, that is `informantName`. So, I guess only handling null value `informantName` should suffice.

**Comment by tareq89** [comment]
> @tareq89 what is the root cause for the issue mentioned in the ticket? What happens if you do not add a helper for formatting an undefined value?
The root cause was, the issue was created on Aug 4, 2023, a long time ago, when if informant name was null, there was no mail. But in time it was fixed I guess but the issue was not addressed.
Later when I picked the issue, I found that it has been solved, other than a cosmetic issue where there is a space in between Hello and comma.

**Comment by tareq89** [comment]
> > @tareq89 what is the root cause for the issue mentioned in the ticket? What happens if you do not add a helper for formatting an undefined value?

 The root cause was, the issue was created on Aug 4, 2023, a long time ago, when if informant name was null, there was no mail. But in time it was fixed I guess but the issue was not addressed. Later when I picked the issue, I found that it has been solved, other than a cosmetic issue where there is a space in between Hello and comma.

**Comment by rikukissa** [comment]
Ok, I understand the issue. Let's add an explicit condition to the templates for this so it's clear by reading it that there is a chance the variable is undefined and we do not need a specific helper for this. Countries might want to also use a different copy for this so it's better to not have a too high-level abstraction

```
{{#if informantName}}
Hello {{informantName}},
{{else}}
Hello,
{{/if}}
```

**Comment by tareq89** [comment]
> Ok, I understand the issue. Let's add an explicit condition to the templates for this so it's clear by reading it that there is a chance the variable is undefined and we do not need a specific helper for this. Countries might want to also use a different copy for this so it's better to not have a too high-level abstraction
> 
> ```
> {{#if informantName}}
> Hello {{informantName}},
> {{else}}
> Hello,
> {{/if}}
> ```

Agreed, on it.

**Review by Zangetsu101 — APPROVED** [review]
The changes look good. Could you just update the title please?
